src/Standards/Squiz/Sniffs/PHP/DisallowMultipleAssignmentsSniff.php

```diff
@@ -47,7 +47,7 @@ public function process(File $phpcsFile, $stackPtr)
         if ($function !== false) {
             $opener = $tokens[$function]['parenthesis_opener'];
             $closer = $tokens[$function]['parenthesis_closer'];
-            if ($opener < $stackPtr && $closer > $stackPtr) {
+            if ($closer === null || ($opener < $stackPtr && $closer > $stackPtr)) {
                 return;
             }
```

Comment:
The code is accessing the `parenthesis_closer` before this condition.

Would it make sense to move the check for a missing parenthesis closer up ?

I.e.:
```suggestion
            if (isset($tokens[$function]['parenthesis_closer']) === false) {
                // Live coding/parse error. Bow out.
                return;
            }

            $opener = $tokens[$function]['parenthesis_opener'];
            $closer = $tokens[$function]['parenthesis_closer'];
            if ($opener < $stackPtr && $closer > $stackPtr) {
                return;
            }
```

Reply:
I implemented it the way I did as `$tokens[$function]['parenthesis_closer']` is always defined. That is why I did not see a problem with the code accessing it before this condition.

That said, in my opinion, your suggestion reads better than what I implemented it. So, I just committed.

I ran into the following error when trying to use the GitHub interface to commit your suggestion, so I manually create a new commit and hopefully used the correct tag to mark you as a coauthor of the commit.

![Screenshot from 2024-07-18 10-30-02](https://github.com/user-attachments/assets/3a2befbc-9142-4501-9738-a503bfee43f3)

This PR is ready for another review.